### PR TITLE
WebGLRenderer render method type definition: accept Object3D as first argument

### LINKED ISSUE #19563: WebGLRenderer render type definition
##### Description of the problem

When using ThreeJS in JavaScript I was able to render only one mesh (without using a Scene) doing:
```javascript
renderer.setRenderTarget(my_target);
renderer.clear();

renderer.render(my_mesh, camera);
```

This is very useful for doing some pre-processing things (in my case, rendering water caustics in shaders using a water mesh as input).

My issue is that I cannot perform the same in TypeScript, the `render` method takes a `Scene` as first argument, so my code does not compile.

Should I open a PR fixing the type definition? I don't see any documentation specifying that my code is valid (that the render method can actually take a mesh as input).

##### Three.js version

r112

### PR DESCRIPTION
**Pull request by martinRenou**
fix #19563

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
@mrdoob This PR seems good to go.

**Comment by mrdoob** [comment]
`Scene` extends `Object3D`.

**Comment by martinRenou** [comment]
What do you mean @mrdoob ?

File: src/renderers/WebGLRenderer.d.ts [hunk 1]
```diff
@@ -350,7 +350,7 @@ export class WebGLRenderer implements Renderer {
 	 * properties to false. To forcibly clear one ore more buffers call {@link WebGLRenderer#clear .clear}.
 	 */
 	render(
-		scene: Scene,
+		scene: Scene | Object3D,
```
Comment (Mugen87) [inline src/renderers/WebGLRenderer.d.ts:353]: Ah, it means it's sufficient to write:
```ts
scene: Object3D,
```

Comment (martinRenou) [reply src/renderers/WebGLRenderer.d.ts:353]: Oh sorry I get it (Friday afternoon, right?). Will do.

**Comment by martinRenou** [comment]
I also noticed the `compile` method which has the same signature as `render`, can it accept any kind of `Object3D` as well? Should I update it?

**Comment by Mugen87** [comment]
I would say yes.

**Comment by martinRenou** [comment]
Done. Please tell me if you need any other change.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**